skore/src/skore/project/open.py

```diff
@@ -0,0 +1,50 @@
+"""Command to open a Project."""
+
+from pathlib import Path
+from typing import Union
+
+from skore.project.create import _create
+from skore.project.load import _load
+from skore.project.project import Project
+
+
+def open(
+    project_path: Union[str, Path] = "project.skore",
+    *,
+    create: bool = True,
+    overwrite: bool = False,
+) -> Project:
+    """Open a project given a project name or path.
+
+    Parameters
+    ----------
+    project_path: Path-like, default is "project.skore"
+        The relative or absolute path of the project.
+    create: bool, default is True
+        Create the project if it does not exist.
+    overwrite: bool, default is False
+        Overwrite the project file if it already exists and ``create`` is ``True``.
+        Has no effect otherwise.
```

Comment:
I would suggest the following but is more a scikit-learn formatting rather than a numpydoc one.

```suggestion
    project_path: Path-like, default="project.skore"
        The relative or absolute path of the project.
    create: bool, default=True
        Create the project if it does not exist.
    overwrite: bool, default=False
        Overwrite the project file if it already exists and ``create`` is ``True``.
        Has no effect otherwise.
```

Reply:
done, yes this looks like the most common formatting in skore